airflow/jobs/triggerer_job_runner.py

```diff
@@ -95,7 +95,7 @@ def __init__(
     ):
         super().__init__(job)
         if capacity is None:
-            self.capacity = conf.getint("triggerer", "default_capacity", fallback=1000)
+            self.capacity = conf.getint("triggerer", "capacity", fallback=1000)
```

Comment:
```suggestion
            self.capacity = conf.getint("triggerer", "capacity")
```

While we are here, let's remove the unnecessary fallback.

Reply:
Done!